int16 yaw_rate
int16 roll_rate
int16 pitch_rate
---
bool result 
